client/styles/Login.module.css

```diff
@@ -0,0 +1,96 @@
+.main {
+  min-height: 90vh;
+  padding: 4rem 0;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+
+.title {
+  margin: 0;
+  line-height: 1.15;
+  font-size: 4rem;
+}
+
+.title a {
+  color: #0070f3;
+  text-decoration: none;
+}
+
+.title,
+.description {
+  text-align: center;
+}
+
+.moodTitle{
+  width: 100%;
+}
+
+.grid {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  flex-wrap: wrap;
+  width: 80%;
+  flex-direction: column;
+}
+
+.moodGrid{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  flex-wrap: wrap;
+  width: 80%;
+  flex-direction: row;
+}
+.brandBox{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* .footer {
```

Comment:
no commented code

Reply:
this is also called `dead code` sometimes